### ISSUE TITLE
[feature] convert $nodes and $properties to lower()

### ISSUE THREAD

**Issue by DanielDecker** [Qu4tro/homie-spec#3]
When creating a device with upper case nodes or a node with upper case properties names, the output of device.messages() will yield them the exactly the same case while the topic path is in lower case:
```python
hearthbeat = Node(
    name="Device hearthbeat",
    typeOf="alive",
    properties={
        'String':Property(name='strinG', formatOf='', datatype=Datatype.STRING, get=None),
        'Num':Property(name='nuM', formatOf='', datatype=Datatype.INTEGER, get=None)
        }
)

desktop = Device(id="Desktop", name="Desktop Computer", nodes={"Local-time": localtime, "Hearthbeat" : hearthbeat, "Temperatur" : temp})

for msg in desktop.messages():
    print(msg.attrs)
```
```
...
{'topic': 'homie/desktop/$nodes', 'payload': 'Local-time,Hearthbeat,Temperatur', 'retained': True, 'qos': 1}
...
{'topic': 'homie/desktop/hearthbeat/$properties', 'payload': 'String,Num', 'retained': True, 'qos': 1}
{'topic': 'homie/desktop/hearthbeat/string/$name', 'payload': 'strinG', 'retained': True, 'qos': 1}
{'topic': 'homie/desktop/hearthbeat/string/$datatype', 'payload': 'string', 'retained': True, 'qos': 1}
{'topic': 'homie/desktop/hearthbeat/string/$format', 'payload': '', 'retained': True, 'qos': 1}
{'topic': 'homie/desktop/hearthbeat/num/$name', 'payload': 'nuM', 'retained': True, 'qos': 1}
{'topic': 'homie/desktop/hearthbeat/num/$datatype', 'payload': 'integer', 'retained': True, 'qos': 1}
{'topic': 'homie/desktop/hearthbeat/num/$format', 'payload': '', 'retained': True, 'qos': 1}
```
I think it would be easier to use homie-spec if the output of $nodes and $properties would be in lower case also.
What do you think about that?

**Comment by Qu4tro** [comment]
That makes sense. The payload for `$nodes` and `$properties` are meant for discoverability and topic ids must be lowercase. Yeah, agreed.

**Comment by DanielDecker** [comment]
Ok, I'll prepare a PR!

**Comment by Qu4tro** [comment]
Fixed in #4 . Thanks @DanielDecker .
